test/models/patient_test.rb

```diff
@@ -3,9 +3,17 @@
 require 'test_case'
 
 class PatientTest < ActiveSupport::TestCase
-  def setup; end
+  def setup
+    @default_purgeable_after = ADMIN_OPTIONS['purgeable_after']
```

Comment:
Can you elaborate on this change? Are you expecting the `ADMIN_OPTIONS` to potentially be changed in these tests such that the teardown needs to reset them?

Reply:
Yes the admin options are manipulated in this test and were causing side-effect errors in the purge job tests.